test/features/parsoid/html2html-bodyOnly/request.json

```diff
@@ -0,0 +1,32 @@
+{
+    "bodyOnly": true,
+    "html": {
+        "headers": {
+            "content-type": "text/html;profile=mediawiki.org/specs/html/1.0.0"
+        },
+        "body": "<html>The modified HTML</html>"
+    },
+    "original": {
```

Comment:
There is no final spec for the public transform end point yet. We were talking about a flat POST API as [in the current Parsoid v1 API](https://www.mediawiki.org/wiki/Parsoid/API), slightly generalized for the `/transform` hierarchy.

Example:

```
POST /{domain}/v1/transform/html/to/wikitext/{title}/{oldid}
Content-type: multipart/form-data

data: '<html>...</html>'
```